ale_linters/d/dls.vim

```diff
@@ -0,0 +1,27 @@
+" Author: aurieh <me@aurieh.me>
+" Description: A Language Server implementation for D
+
+call ale#Set('d_dls_executable', 'dls')
+
+function! ale_linters#d#dls#GetExecutable(buffer) abort
+    return ale#Var(a:buffer, 'd_dls_executable')
+endfunction
+
+function! ale_linters#d#dls#GetLanguage(buffer) abort
+    return 'd'
+endfunction
```

Comment:
You can remove this function and `language_callback` below. The language given to the `Define` function will be used by default.

Reply:
Yeah, I'm used to the old way. Will change in a sec.